cpp/src/strings/convert/convert_integers.cu

```diff
@@ -102,19 +202,18 @@ std::unique_ptr<column> to_integers(strings_column_view const& strings,
 {
   size_type strings_count = strings.size();
   if (strings_count == 0) return make_numeric_column(output_type, 0);
-  auto strings_column = column_device_view::create(strings.parent(), stream);
-  auto d_strings      = *strings_column;
-  // create integer output column copying the strings null-mask
-  auto results      = make_numeric_column(output_type,
+
+  // Create integer output column copying the strings null-mask
+  auto results = make_numeric_column(output_type,
                                      strings_count,
                                      cudf::detail::copy_bitmask(strings.parent(), stream, mr),
                                      strings.null_count(),
                                      stream,
                                      mr);
-  auto results_view = results->mutable_view();
-  // fill output column with integers
-  type_dispatcher(output_type, dispatch_to_integers_fn{}, d_strings, results_view, stream);
-  results->set_null_count(strings.null_count());
```

Comment:
This line should not have been removed.

Reply:
I see that in the line above that line:
```
 auto results = make_numeric_column(data_type{type_id::BOOL8},
                                       strings.size(),
                                       cudf::detail::copy_bitmask(input.parent(), stream, mr),
                                       strings.null_count(),
                                       stream,
                                       mr);
```
, the nullmask and null_count have been copied over to the results column, thus we don't need to assign that anymore.

Reply:
Actually, calling `mutable_view()` on a `column` invalidates it's null count. This will set it back and save an extra kernel call for anyone  using the output column downstream.

Reply:
Thanks for this. It is not an obvious "feature". I added those line back and also added comment for those line to clarify the purpose.